--- ../CMakeLists.txt	2020-06-08 11:24:23.000000000 -0400
+++ CMakeLists.txt	2020-11-02 21:24:43.405111521 -0500
@@ -74,13 +74,8 @@
 # OpenTracingConfig.cmake file
 find_package(OpenTracing CONFIG REQUIRED)
 # Under Windows, link dynamically with OpenTracing
-if (WIN32)
-  list(APPEND LIBS OpenTracing::opentracing)
-  set(OPENTRACING_LIB OpenTracing::opentracing)
-else()
-  list(APPEND LIBS OpenTracing::opentracing-static)
-  set(OPENTRACING_LIB OpenTracing::opentracing-static)
-endif()
+list(APPEND LIBS OpenTracing::opentracing)
+set(OPENTRACING_LIB OpenTracing::opentracing)
 list(APPEND package_deps OpenTracing)
 
 hunter_add_package(nlohmann_json)
